WalletWasabi/Hwi/ProcessBridge/HwiProcessBridge.cs

```diff
@@ -19,9 +19,9 @@ public HwiProcessBridge() : base(MicroserviceHelpers.GetBinaryPath("hwi"))
 		{
 		}
 
-		public new async Task<(string response, int exitCode)> SendCommandAsync(string arguments, bool openConsole, CancellationToken cancel)
+		public new async Task<(string response, int exitCode)> SendCommandAsync(string arguments, bool openConsole, CancellationToken cancel, Action<StreamWriter> standartInputWriter = null)
```

Comment:
This is wrong and it works because planets align. The `new` modifier is shadowing the `base` implementation and i remember this was not called in some situations.

Reply:
Strange it should work. The new keyword hiding the inherited member. Can you try to figure out a failing test for it?

Reply:
the only difference is when open-console is true. I am adding one test to test if the call is OK.

Reply:
I might found a bug here. If you set openconsole you cannot redirect

System.InvalidOperationException : The Process object must have the UseShellExecute property set to false in order to redirect IO streams.

Reply:
I am as much flexible as possible with the reviews at the point that I could be okay with nested `goto` statements but this `new` modifier should not be here, there is no reason to have it and it is in general a very bad idea.

Reply:
`New` keyword seems to be the perfect fit here. HwiProcessBridge inherited from ProcessBridge as we do with other process bridges. Also, HwiProcessBridge must confirm with IProcessBridge as it is passed to the HwiClient constructor. 

So the other way to remove `new` is to break the inheritance and use ProcessBridge as a class member in HwiProcessBridge.

What is your exact problem with new?